ui/src/components/MessageEditor.tsx

```diff
@@ -93,13 +93,6 @@ export function useMessageEditor({
       if (!whom) {
         return false;
       }
-      const text = event.clipboardData?.getData('text/plain');
-
-      if (text && isRef(text)) {
```

Comment:
If the pasted item is a ref, is it still handled as expected without this conditional branch?

Reply:
Yes, this was just a way to force it since it wasn't happening using the existing method on Android.